### PR TITLE
Use pip3 to install checkov without target

### PR DESCRIPTION
**Pull request by elaygl**
**In This PR**
- Use `pip3` without specifying a target

- [X] I've reviewed my own code

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { CheckovInstallation, installOrUpdateCheckov } from './checkovInstaller'
 import { runCheckovScan } from './checkovRunner';
 import { applyDiagnostics } from './diagnostics';
 import { fixCodeActionProvider, providedCodeActionKinds } from './suggestFix';
-import { getLogger, saveCheckovResult, isSupportedFileType, extensionVersion, escapePath } from './utils';
```
Comment (SteveVaknin) [inline src/extension.ts:9]: I think you can also remove the `escapePath` and `unescapePath` from the utils

File: src/checkovRunner.ts [hunk 2]
```diff
@@ -35,12 +34,11 @@ interface CheckovResponseRaw {
...
 const skipChecks = ['CKV_AWS_52'];
 
-export const runCheckovScan = (logger: Logger, checkovPath: string, extensionVersion: string, fileName: string, token: string, cancelToken: vscode.CancellationToken): Promise<CheckovResponse> => {
+export const runCheckovScan = (logger: Logger, extensionVersion: string, fileName: string, token: string, cancelToken: vscode.CancellationToken): Promise<CheckovResponse> => {
     return new Promise((resolve, reject) => {
         const checkovArguments: string[] = ['-s', '--skip-check', skipChecks.join(','), '--bc-api-key', token, '--repo-id', 'vscode/extension', '-f', `"${fileName}"`, '-o', 'json'];
-        const checkovCommand = `"${unescapePath(checkovPath)}"`;
-        logger.info('Running checkov', { checkovCommand, arguments: checkovArguments });
-        const ckv = spawn(checkovCommand, checkovArguments, 
+        logger.info('Running checkov', { arguments: checkovArguments });
```
Comment (SteveVaknin) [inline src/checkovRunner.ts:40]: Now that I think about it, logging the token isn't a great practice...especially if we want them to share their log files in issues or slack

This is the most straight forward way

```
        const checkovLogArguments: string[] = ['-s', '--skip-check', skipChecks.join(','), '--bc-api-key', '****', '--repo-id', 'vscode/extension', '-f', `"${fileName}"`, '-o', 'json'];
        logger.info('Running checkov', { arguments: checkovLogArguments });
```
We can also map and change the token only 

```
        logger.info('Running checkov', { arguments: checkovLogArguments.map(argument => argument === token ? '****' : argument });
```

@elaygl WDYT?

**Review by SteveVaknin — APPROVED** [review]
Looks good, Left minor comments

File: src/extension.ts (hunk 1 above)
Comment (elaygl) [reply src/extension.ts:9]: Good catch

File: src/checkovRunner.ts (hunk 2 above)
Comment (elaygl) [reply src/checkovRunner.ts]: Sounds good

File: src/checkovInstaller.ts [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
-import { join } from 'path';
 import { asyncExec } from './utils';
 
-const updateCheckovWithSystemPython = async (logger: Logger, installInPath: string): Promise<string> => {
+const updateCheckovWithPip3 = async (logger: Logger): Promise<void> => {
     try {
         logger.info('Trying to install Checkov using pip3.');
-        await asyncExec(`pip3 install -U --target ${installInPath} checkov -i https://pypi.org/simple/`);
-        return join(installInPath, 'bin', 'checkov');
+        await asyncExec('pip3 install -U checkov -i https://pypi.org/simple/');
```
Comment (schosterbarak) [inline src/checkovInstaller.ts:7]: @elaygl maybe we should add --verbose or --log for debugging purposes

**Pull request merged**